### PR TITLE
7 implement a body class

### LINKED ISSUE #7: Implement a body class
Class Body:
   ...
   ...

me = Body(10, [x,y,z])
me.mass => 10

### PR DESCRIPTION
**Pull request by 0610Thomas**
Made a body class where there is mass and position

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -49,7 +56,9 @@ def force(mass: float, acc: Vector) -> Vector:
 
 mass = 10.0
 accel = Vector(2,3,1)
```
Comment (tonypierre) [inline main.py:58]: accel is not being used

Comment (0610Thomas) [reply main.py:58]: We are going to use it later

Comment (tonypierre) [reply main.py:58]: Okay lgtm

**Review by tonypierre — APPROVED** [review]
LGTM

**Pull request merged**